pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -14,17 +14,27 @@ type HyperConvergedSpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	KWebUIMasterDefaultSubdomain string `json:"KWebUIMasterDefaultSubdomain,omitempty"` // optional, required if openshift-console project is missing
-	KWebUIPublicMasterHostname   string `json:"KWebUIPublicMasterHostname,omitempty"` // optional, required if openshift-console project is missing
+	KWebUIPublicMasterHostname   string `json:"KWebUIPublicMasterHostname,omitempty"`   // optional, required if openshift-console project is missing
 }
 
+// HyperConvergedPhase is a label for the phase of a HCO deployment at the current time.
+// ---
+// +k8s:openapi-gen=true
+type HyperConvergedPhase string
```

Comment:
I'm not sure we need `+k8s:openapi-gen=true` because openapi-gen will catch this string in `type HyperConvergedStatus struct`.